src/entity/units/golem.ts

```diff
@@ -14,14 +14,14 @@ const unit: UnitSource = {
     subtype: UnitSubType.MELEE,
   },
   unitProps: {
-    damage: 30,
-    staminaMax: config.UNIT_BASE_STAMINA,
-    healthMax: 20,
+    damage: 20,
+    staminaMax: config.PLAYER_BASE_ATTACK_RANGE,
+    healthMax: 40,
     manaMax: 0,
   },
   spawnParams: {
-    probability: 100,
-    budgetCost: 1,
+    probability: 10,
```

Comment:
90% reduction in probability for the core basic unit??

Reply:
Yes, the goal is less cannon fodder in later levels. It's more likely to pick non-basic golem/archer/glop.